docs/api-reference/next/image.md

```diff
@@ -14,182 +14,222 @@ description: Enable Image Optimization with the built-in Image component.
+| `v12.2.0` | Experimental `remotePatterns` and experimental `unoptimized` configuration added. `layout="raw"` removed.                                                                                                                                                            |
+| `v12.1.1` | `style` prop added. Experimental[\*](#experimental-raw-layout-mode) support for `layout="raw"` added.                                                                                                                                                                |
+| `v12.1.0` | `dangerouslyAllowSVG` and `contentSecurityPolicy` configuration added.                                                                                                                                                                                               |
+| `v12.0.9` | `lazyRoot` prop added.                                                                                                                                                                                                                                               |
+| `v12.0.0` | `formats` configuration added.<br/>AVIF support added.<br/>Wrapper `<div>` changed to `<span>`.                                                                                                                                                                      |
+| `v11.1.0` | `onLoadingComplete` and `lazyBoundary` props added.                                                                                                                                                                                                                  |
+| `v11.0.0` | `src` prop support for static import.<br/>`placeholder` prop added.<br/>`blurDataURL` prop added.                                                                                                                                                                    |
+| `v10.0.5` | `loader` prop added.                                                                                                                                                                                                                                                 |
+| `v10.0.1` | `layout` prop added.                                                                                                                                                                                                                                                 |
+| `v10.0.0` | `next/image` introduced.                                                                                                                                                                                                                                             |
```

Comment:
Whats changing in this table? I wouldnt expect this to change?

Reply:
I added the top line, I think it's helpful to have.